Gems/Atom/Feature/Common/Assets/Shaders/Materials/MaterialInputs/SilhouetteBlockerInput.azsli

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+option bool o_blockSilhouettes;
```

Comment:
Where is the shader option used in a shader?

Reply:
It isn't, this is an unfortunate quirk with the material property groups, if you want to turn something on and off you have to have a shader variable even if it isn't used.  This is used by SilhouetteBlockerPropertyGroup.json